### ISSUE TITLE
Replace flag doesn't work on RBAC-enabled clusters

### ISSUE THREAD

**Issue by stepro** [stepro/kdo#37]
When a cluster is RBAC-enabled, such that service accounts don't (by default) have privileges to do anything with the Kubernetes API, the replace flag doesn't work properly because the kdo-replacer container that is injected into the pod tries to use kubectl to scale replicas or update service selectors and this fails.

It seems like replacement will need to be done by the client, and reverting will need to be handled by the cluster-wide server components (which will also need to be installed with an appropriate server admin service account).

**Comment by stepro** [comment]
The workaround for the getting started tutorial is to run this:

```
kubectl create clusterrolebinding kdo-todo-app --clusterrole=cluster-admin --serviceaccount=kdo-todo-app:default
```

**Comment by stepro** [comment]
Resolved by #39.
